css/properties/display.json

```
@@ -4,6 +4,7 @@
       "display": {
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/display",
+          "spec_url": "https://drafts.csswg.org/css-flexbox-1/#flex-containers",
```

This is the main display feature, I think only sub features should talk about flex?

```suggestion
          "spec_url": "https://drafts.csswg.org/css-display/#propdef-display",
```